### PR TITLE
Added possibility to create collection with filters in url or filters model.

### PR DESCRIPTION
**Pull request by stepek**

### CODE REVIEW

File: app/js/models/schemaModel.js [hunk 1]
```diff
@@ -322,6 +323,22 @@ export default class SchemaModel extends Model {
       options = Object.assign({}, {pageLimit: options});
     }
 
+    if (/\?/.test(url)) {
+      let filtersString = url.match(/(\?.*)$/g)[0].slice(1);
+      let filtersArray = filtersString.split('&');
+
+      options.filters = Object.assign({}, options.filters);
+
+      for (let item of filtersArray) {
```
Comment (stepek) [inline app/js/models/schemaModel.js]: 'for of' and 'forEach' works similarly...

Comment (stepek) [inline app/js/models/schemaModel.js]: No.
eg: https://github.com/cloudwan/gohan_webui/blob/master/app/js/models/schemaModel.js#L988

**Comment by thayashi** [comment]
Suppose I'd like to have a default collection and a filtered collection like:
https://10.15.2.155:9999/v1.0/peer_sessions
https://10.15.2.155:9999/v1.0/peer_sessions?local_ese_device_id=4d7db5d1-af42-4936-955d-06992d114230
But this.url will be same as `https://10.15.2.155:9999/v1.0/peer_sessions` in either case when it is set to `self.collections`.
https://github.com/stepek/gohan_webui/blob/00ae472771ed26595aa262b4d15812dbde0900bb/app/js/models/schemaModel.js#L665
So in the above case, the second one returns first collection not creating new one.
May be it needs to call updateUrl() before set to `self.collections`?

**Comment by stepek** [comment]
Fixed

**Comment by thayashi** [comment]
If I set the filter by parameter for makeCollection like this:
```
    const collection = schema.makeCollection(undefined, {
      pageLimit: 25,
      filters: {
        local_ese_device_id: id
      }
    }); 
```
it doesn't reflect to `resourceUrl`, same problem will happen. So resourceUrl should be generated from options.filters (after added values from url).

**Comment by thayashi** [comment]
LGTM

**Pull request merged**